src/middlewared/middlewared/plugins/device_/device_info_base.py

```diff
@@ -43,7 +43,7 @@ async def get_serials(self):
         raise NotImplementedError()
 
     @private
```

Comment:
We don't need this files separation anymore, can you please merge them two into just `device_info.py`?

Reply:
Yep, no problem will do